tutor/specs/e2e/my-courses.e2e.ts

```diff
@@ -12,56 +11,114 @@ describe('My Courses', () => {
+            // //preview card
+            await expect(page).toHaveSelector('testEl=preview-course-card')
+            // //create course card
+            await expect(page).toHaveSelector('testEl=create-course')
+    
+            //Resources
+            await page.click('testEl=resources-tab')
+            await expect(page).toHaveText('testEl=getting-started-guide', 'Instructor Getting Started Guide')
+            await expect(page).toHaveText('testEl=getting-started-video', ' Video Tutorials ')
         })
-        await visitPage(page, '/courses')
-        await page.click('.my-courses-item.preview a')
-        await page.waitForNavigation()
-        expect(
-            await page.evaluate(() => window.location.pathname)
-        ).toMatch(/course\/\d+/)
-    })
+    
+        it('enters edit mode', async () => {
+            const getOfferingIds = () => {
+                return Array.from(document.querySelectorAll<HTMLDivElement>('.offering-container'))
+                    .reduce((current, el) => el.dataset?.offeringId 
+                        ? current.concat(el.dataset.offeringId) : current, [] as string[])
+            }
+
+            mock.current.bootstrapData.courses.forEach((c: any, index: number) => {
+                //grouping courses with the first two offerings, id 1 and id 2
+                c.offering_id = mock.current.bootstrapData.offerings[index % 2].id
+            })
+            await visitPage(page, '/courses')
+            await expect(page).toHaveSelector('testEl=existing-teacher-screen')
+
+            await page.click('.controls button')
+            await expect(await page.$eval('.controls button', node => node.textContent)).toEqual('Exit settings')
+            await expect(page).toHaveSelector('.edit-mode-icons')
+            await expect(page).toHaveSelector('.add-subject-dropdown')
 
-    it('hides concept coach courses', async () => {
-        const c = mock.current.course(1)
-        c.is_concept_coach = true
-        c.name = 'ConceptCoach'
-        await visitPage(page, '/courses')
-        await expect(page).toHaveSelector('testEl=existing-teacher-screen')
-        await expect(page).not.toHaveText('testEl=course-card', 'ConceptCoach', { timeout: 100 })
+            // [1, 2]
+            const offeringOrderIds = await page.evaluate(getOfferingIds)
+            // ordering the offerings
+            await page.click(`.offering-container[data-offering-id="${offeringOrderIds[0]}"] .ox-icon-arrow-down`)
+            await expect(await page.evaluate(getOfferingIds)).toEqual(offeringOrderIds.reverse())
+
+            const defaultOfferingContainersLength = await page.$$eval('.offering-container', node => node.length)
+            await page.click('.add-subject-dropdown button')
+            await page.waitForSelector('.dropdown-menu.show')
+            await page.click('.dropdown-menu.show .offering-item:not(.disabled)', { timeout: 500 })
```

Comment:
the only reason you need page.waitForSelector is because you've set a timeout for clicking it.  Pretty sure this could be:
```ts
await page.click('.add-subject-dropdown button')
await page.click('.dropdown-menu.show .offering-item:not(.disabled)')

```